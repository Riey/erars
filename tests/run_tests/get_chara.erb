@SYSTEM_TITLE
ADDCHARA 3
GETCHARA 3
PRINTFORM GETCHARA 3 = {RESULT}, GETCHARA(3) = {GETCHARA(3)}
