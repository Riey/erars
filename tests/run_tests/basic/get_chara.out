GETCHARA 3 = 0, GETCHARA(3) = 0